web/src/Room.tsx

```diff
@@ -516,9 +547,10 @@ export default function Room() {
             className="flex rounded-md shadow-sm"
             onSubmit={(e) => {
               e.preventDefault();
+              const slug = sanitizeSlug(addToQueueInput);
```

Comment:
It's only ever used once, and it doesn't have independent tests, so I'd probably just inline this.

Reply:
Went a bit overboard learning new syntax LOL